napari/utils/colormaps/colormaps.py

```diff
@@ -265,6 +301,7 @@ def vispy_or_mpl_colormap(name):
     vispy_cmaps = get_colormaps()
     if name in vispy_cmaps:
         cmap = get_colormap(name)
+        cmap = convert_vispy_colormap(cmap, name=name)
```

Comment:
I think the docstring needs to be updated now that the returned colormap is a napari `Colormap`.